plugins/talk-plugin-moderation-actions/client/reducer.js

```diff
@@ -1,7 +1,9 @@
-import {OPEN_MENU, CLOSE_MENU} from './constants';
+import {OPEN_MENU, CLOSE_MENU, OPEN_DIALOG, CLOSE_DIALOG} from './constants';
 
 const initialState = {
   showMenuForComment: null,
+  showDialog: false,
```

Comment:
Should be called `showBanDialog`, as we might also have a `showSuspendDialog` later?